xbmc/interfaces/legacy/InfoTagPicture.h

```diff
@@ -0,0 +1,176 @@
+/*
+ *  Copyright (C) 2021 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "AddonClass.h"
+
+class CPictureInfoTag;
+
+namespace XBMCAddon
+{
+namespace xbmc
+{
+
+///
+/// \defgroup python_InfoTagPicture InfoTagPicture
+/// \ingroup python_xbmc
+/// @{
+/// @brief **Kodi's picture info tag class.**
+///
+/// \python_class{ InfoTagPicture() }
+///
+/// Access and / or modify the picture metadata of a ListItem.
+///
+///-------------------------------------------------------------------------
+///
+/// **Example:**
+/// ~~~~~~~~~~~~~{.py}
+/// ...
+/// tag = item.getPictureInfoTag()
+///
+/// datetime_taken  = tag.getDateTimeTaken()
+/// tag.setResolution(1920, 1080)
+/// ...
+/// ~~~~~~~~~~~~~
+///
+class InfoTagPicture : public AddonClass
+{
+private:
+  CPictureInfoTag* infoTag;
+  bool offscreen;
+  bool owned;
+
+public:
+#ifndef SWIG
+  explicit InfoTagPicture(const CPictureInfoTag* tag);
+  explicit InfoTagPicture(CPictureInfoTag* tag, bool offscreen = false);
+#endif
+
+#ifdef DOXYGEN_SHOULD_USE_THIS
+  ///
+  /// \ingroup python_InfoTagPicture
+  /// @brief \python_func{ xbmc.InfoTagPicture([offscreen]) }
+  /// Create a picture info tag.
+  ///
+  /// @param offscreen            [opt] bool (default `False`) - if GUI based locks should be
```

Comment:
Wondering if it's not a good idea to default offscreeen to true since these are new methods. It wasn't made enabled by default in listitem just for backward compatibility reasons afaik

Reply:
usually the situation is this when you need to set the info:

```python
list_item = ListItem(offscreen=True)
info = list_item.getVideoInfoTag()  #<<<at this point the offscreen is already set
info.setPlot("bla")
...
```

then also if you change to true in the object constructor doesn't make difference...

Instead if we want to make it actually True as default,
we will need to create a new argument as True like "isOffscreen" and deprecate the old one,
this could act as a transition for addons
It would be very useful because I have noticed that many addons ignore offscreen argument and avoid setting it as True